src/styles/components/stars.scss

```diff
@@ -0,0 +1,30 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+  align-items: flex-end;
+
+  &__list {
+    display: flex;
+    gap: 4px;
+    padding: 0;
+    margin: 0;
+    justify-content: center;
+  }
+
+  &__item:not(:last-child) {
```

Comment:
Consider using the CSS from the previous task to keep classes with modifiers working